### PR TITLE
magento/devdocs#: Add createEmptyCart mutation API errors

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) adds information about API errors of `createEmptyCart` mutation.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/graphql/mutations/create-empty-cart.html

## Links to Magento source code

-  [Cart ID length should to be 32 symbols.](https://github.com/magento/magento2/blob/2.3.3/app/code/Magento/QuoteGraphQl/Model/Resolver/CreateEmptyCart.php#L84)
- [Cart with ID "%1" already exists.](https://github.com/magento/magento2/blob/2.3.3/app/code/Magento/QuoteGraphQl/Model/Resolver/CreateEmptyCart.php#L88)

Thank you!

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.3/graphql/mutations/create-empty-cart.md [hunk 1]
```diff
@@ -71,3 +71,10 @@ Attribute |  Data Type | Description
 ## Output attributes
 
 The `createEmptyCart` mutation returns the cart ID.
+
+## Errors
+
+Error | Description
+--- | ---
+`Cart with ID "XXX" already exists.` | Cart with provided ID already created.
```
Comment (keharper) [inline guides/v2.3/graphql/mutations/create-empty-cart.md]:
```suggestion
`Cart with ID "XXX" already exists.` | The specified cart ID was previously used to create a cart.
```

File: guides/v2.3/graphql/mutations/create-empty-cart.md [hunk 2]
```diff
@@ -71,3 +71,10 @@ Attribute |  Data Type | Description
...
+`Cart ID length should to be 32 symbols.` | The length of Cart ID is not equal to required length.
```
Comment (keharper) [inline guides/v2.3/graphql/mutations/create-empty-cart.md]:
```suggestion
`Cart ID length should to be 32 symbols.` | The cart ID is not the required length.
```

File: guides/v2.3/graphql/mutations/create-empty-cart.md (hunk 1 above)
Comment (atwixfirster) [reply guides/v2.3/graphql/mutations/create-empty-cart.md]: **done**

File: guides/v2.3/graphql/mutations/create-empty-cart.md (hunk 2 above)
Comment (atwixfirster) [reply guides/v2.3/graphql/mutations/create-empty-cart.md]: **done**

**Comment by atwixfirster** [comment]
Thanks, @keharper

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
running tests

**Pull request merged**